src/oauth2.js

```diff
@@ -2,10 +2,24 @@ import querystring from 'query-string'
 import cuid from 'cuid'
 import openPopup from './util/popup'
 
-const listenForCredentials = (popup, state, resolve, reject) => {
+const listenForCredentials = (popup, state, resolve, reject, config) => {
```

Comment:
I would rather have the `config` param as the first one.

Reply:
Aye sir. Will move it up to the first.